EIPS/eip-2225.md

```diff
@@ -0,0 +1,125 @@
+  mergee chain's fork height block's state root. The same for mergee
+  chain's fork height block.
+* After mergee chain's fork height block, its proof of work should be
+  abandoned. That is, updated miners should not continue to mine
+  mergee chain any more.
+  
+### State Root
+  
+For the block after merger chain's fork height block, the state root
+meaning is changed to be hash of concatenating merger chain's original
+state root, and mergee chain's original state root.
+
+### Block Rewards
+
+Use merger chain's original block rewards. And mine coinbase always to
+the merger chain's side of the state.
+
+### Transactions and Account Versioning
+
+After merge, transactions with both chain ids are considered valid on
+the merger chain. We define two account versions, one with merger
+chain's original EVM config, and one with mergee chain's original EVM
+config. Accessing accounts originally from either chain should use
+EIP-1702's account versioning rules. We define how accounts are
+referred on the merged blockchain below:
+
+* For end user interface, prefix `0x01` for accounts on the merger
+  chain, and prefix `0x02` for accounts on the mergee chain.
+* The transaction format is unchanged with addresses using original
+  format. We distinguish whether a transaction operates on merger chain
+  or mergee chain by its chain id. None replay protected transactions
+  must be disabled prior to merging.
+* All original opcodes `CALL*`, `CREATE*` are unchanged, where it only
+  accesses accounts on the same originating chain.
+  
+### Cross the Merger and Mergee Chain Boundary
+
+Define factor `P // Q`, which determines how coins are exchanged on
+merger chain and mergee chain. Define three new opcodes `MCALL`,
+`MCREATE` and `MCREATE2`. They function the same as `CREATE`, `CALL`
+and `CREATE2` with the same gas cost, except that it calls accounts on
+the other side of the state. Values `V` to be exchanged are computed as
+`V * P // Q` from merger side of state to mergee side of state, and
+`V * Q // P` from mergee side of state to merger side of state.
+
+## Rationale
+
+This EIP accomplishes the chain merge by concatenating two blockchain's
+state root. In clients, this can be represented as a simple depth one
+binary merkle tree where root's left is the merger state, and root's
+right is the mergee state. We use account versioning and transaction
+replay protection to make it backward compatible for full nodes,
+without changing any internal address representation or transaction
+representation. We then define additional opcodes to make two sides
+communicatable.
+
+## Backwards Compatibility
+
+This EIP is fully backward compatible for EVM and account states. It
+is not backward compatible for light client state proof verification,
+because an additional binary merkle tree is added.
+
+## Test Cases
+
+To be added.
+
+## Implementation
+
+To be added.
+
+## Copyright
+
+This work is licensed under a [Creative Commons Attribution-ShareAlike
```

Comment:
EIPs must be in the public domain.  Share Alike is not publc domain.

> Copyright Waiver - All EIPs must be in the public domain. See the bottom of this EIP for an example copyright waiver.

Reply:
Please avoid double standards — we already have merged EIPs with CC-BY license. And I have no intention to change license of this document.

Reply:
Past performance is no guarantee of future behavior.  The requirement of EIP-1 as it is presently constituted remains.

Reply:
Okay sure. But it really only affects this EIP repo — it’s just a publishing platform, by not keeping license requirements consistent we’re damaging its credibility. There’s also nothing preventing me from choosing another publishing platform, if I find audiences being restricted.

Anyway I’m happy to just keep this document as open PR. But I refuses to change the license because 1) it’s double standard, and 2) CC-BY is already a free license.

Reply:
There are two EIPs that were mistakenly published with share alike, 1355, a tweak on FNV for mining and 1485, a tweak on 1355.  1355 is withdrawn, 1485 won't be implemented.  Two others have the spec under CC0 but mention that referred links have other licenses.  1057, and 1592.  Those are OK as the spec itself is is CC0.

Each and every other of the 200+ EIPs either don't discuss copyright or assign CC0, I know I just reviewed them by hand.  And the CC0 only covers the spec, not the reference implementations.

You are right, you can publish this anywhere you want.  But an EIP, even as a draft, has a certain air of credibility.  As this pertains to ethereum chains I do think this is a good place to publish it.  But the editors have set standards for what should be published.